dashboard/README.md

```diff
@@ -79,6 +79,14 @@ pretty_url: "http://user.o6s.io/function"
 
 Set `public_url` to be the URL for the IP / DNS of the OpenFaaS Cloud.
 
+Set `cookie_root_domain` when using auth.
```

Comment:
What about when working locally?

Reply:
If you are deploying it locally, informations from `auth/README.md` applies, and when you assign a domain to your localhost via `/etc/hosts` and you want to test auth locally, then you need to add that assigned domain here also.